sharepoint/sharepoint-ps/sharepoint-online/New-SPOContainerType.md

```diff
@@ -0,0 +1,215 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/new-spocontainertype
+applicable: SharePoint Online
+title: New-SPOContainerType
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: ShreyasSar26
+ms.reviewer:
+---
+
+# New-SPOContainerType
+
+## SYNOPSIS
+
+This cmdlet creates a new containertype of standard or trial status. The standard containertype can be created with the regular billing structure or direct to customer billing structure.
```

Comment:
>containertype [](http://example.com/codeflow?start=26&length=13)

(here and other files)

should this be "container type"? since "containertype" is not a word

Reply:
or should it be "ContainerType"?